library/headers.inc.php

```diff
@@ -37,4 +37,5 @@ function include_js_library($path)
 ?>
 <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/library/textformat.js"></script>
```

Comment:
@nileshprasad137 
maybe a nice switch/case instead of IF statements, or anything clear and compact.  I think one function that lets you have one little bit of documentation for the function arguments, then we can use a (large) single-line function to call most any set of headers we need.